functional-test/src/main/java/org/zanata/page/WebDriverFactory.java

```diff
@@ -60,33 +55,47 @@
 public enum WebDriverFactory {
     INSTANCE;
 
-    private volatile WebDriver driver = createDriver();
+    private volatile EventFiringWebDriver driver = createDriver();
     private DriverService driverService;
     private TestEventForScreenshotListener eventListener;
     private int webdriverWait;
+
     public WebDriver getDriver() {
         return driver;
     }
 
-    private WebDriver createDriver() {
-        WebDriver driver = createPlainDriver();
+    private EventFiringWebDriver createDriver() {
+        String driverType = PropertiesHolder.getProperty(webDriverType.value());
+        EventFiringWebDriver newDriver;
+        switch (driverType.toLowerCase()) {
+            case "chrome":
```

Comment:
This is not big deal, but since we've got `Constants` in FT, perhaps we should move `chrome` and `firefox` to there.